docs/guide/slots.md

```diff
@@ -6,13 +6,17 @@ parent: Building ViewComponents
 
 # Slots
 
-In addition to the `content` accessor, ViewComponents accept content through slots, enabling multiple blocks of content to be passed to a single ViewComponent.
+In addition to the `content` accessor, ViewComponents accept content through slots. Slots are dedicated areas of a
+component that render content, including other components.
 
 Slots are defined with `renders_one` and `renders_many`:
 
 - `renders_one` defines a slot that will be rendered at most once per component: `renders_one :header`
 - `renders_many` defines a slot that can be rendered multiple times per-component: `renders_many :posts`
 
+If you don't specify a second argument to these methods, you'll create a **passthrough slot**. Arbitrary content can be
+rendered inside these slots. If you need to render arbitrary markup or components, these may be the way to go.
+
 For example:
```

Comment:
```suggestion
Arbitrary content can then be passed to the slots when rendering the component:
```

Reply:
I think it's important that we keep the term "passthrough slot" in our glossary. I think it'll make it easier for users of the library to communicate solutions to problems. 

If the naming is an issue, it's just hit me that "content slot" could be a potential alternative here.

Reply:
Maybe we remove the term `arbitrary` then? I'm wondering whether the term `passthrough` is useful for helping people understand this feature or not. I recognize that I am not a good judge of this question as the creator of the framework and thus have different context than most people.

```suggestion
Content can then be passed to the slots when rendering the component:
```

Reply:
I think you're right on both counts - arbitrary feels too vague, and passthrough accurately describes how the slot functions but doesn't fully demonstrate its value.

I've done some more thinking around naming here and I feel like I'm getting close to a potential solution:

- `template slots` is one option. It's trying to convey that this is a slot that forms part of a template (in the dictionary definition sense, not the view file sense). The fact that I had to clarify there is a bit of a red flag, but I feel like folks will reach for the dictionary definition first and thus understand that this is a slot into which anything can be rendered. 
- `content slots` is another, but then again, `content` is practically a (whatever-we're-calling-this) slot in its own right. Whether that causes confusion or actually bolsters understanding of the feature is probably dependent on the reader, but my gut feeling says it'll cause more confusion.

Reply:
I've updated this section as follows:

> If you don't specify a second argument to these methods, you'll create a
**passthrough slot**. Any content you pass through can be rendered inside these
slots. If you don't want to strictly specify what should go in the slot you're
making, these may be the way to go.

How does that sound?

Reply:
And on reflection, I think the naming conflicts with those alternatives I suggested are too strong for them to be considered.